tests/zstreamtest.c

```diff
@@ -1071,19 +1071,19 @@ static int basicUnitTests(U32 seed, double compressibility)
     }
     DISPLAYLEVEL(3, "OK \n");
 
-    DISPLAYLEVEL(3, "test%3i : ZSTD_resetDStream() with dictionary : ", testNb++);
```

Comment:
I would recommend not modifying the test functions,
such as the ones include in `tests/zstreamtest.c`
which scope _includes_ checking static symbols.